### ISSUE TITLE
ImportError issue in __init__.py for Context

### ISSUE THREAD

**Issue by azharem** [if-then-fund/django-html-emailer#5]
__init__.py file imports Context from django.temmplate.base module. However in django 3.1.4 version the Context class is found in djando.template.context module.  Change the import from 

```python
from django.template.base import Template, Context
```

to 

```python
from django.template.base import Template
from django.template.context import Context
```

`__init__.py` file.

I've made this fix in a forked repository branch and [which can be found here](https://github.com/azharem/django-html-emailer/tree/fix/Context_import_error).

**Comment by t-s** [comment]
Note: we should double check compatibility with Django 2.x, and ideally note when compatibility with `django-html-emailer` and `django` broke

**Comment by JoshData** [comment]
Fixed by #6. Thanks all.
